app/views/teaching_events/show.html.erb

```diff
@@ -22,27 +22,19 @@
     </div>
   </header>
 
-  <section class="event-info">
-    <% if @event.message %>
-      <div class="event-alert">
-        <%= tag.p(@event.message) %>
-      </div>
-    <% end %>
+  <div class="container teaching-event__info">
+    <article>
+      <%= render(partial: "teaching_events/show/event-summary") %>
+      <%= render(partial: "teaching_events/show/attending-training-providers") %>
+      <%= render(partial: "teaching_events/show/scribble") if @event.scribble_id.present? %>
+      <%= render(partial: "teaching_events/show/how-to-attend") if is_a_train_to_teach_event?(@event) %>
+      <%= render(partial: "teaching_events/show/provider-info") if @event.show_provider_information? %>
+    </article>
+
+    <aside>
+      <%= render(partial: "teaching_events/show/venue-information") if @event.building.present? %>
```

Comment:
I'm not sure if the behaviour change here is intentional or not, but we currently don't show venue information for 'virtual events' -- a virtual event is online and can (or always, not sure) have a building:

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/events/show.html.erb#L42